src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -509,6 +515,117 @@ export default class AircraftCommander {
 
     /**
      * @for AircraftCommander
+     * @method runSayAltitude
+     * @param aircraft
+     * @return {array} [success of operation, readback]
+     */
+    runSayAltitude(aircraft) {
+        const altitude = _round(aircraft.altitude, -2);
+        const isClimbingOrDescending = aircraft.trend !== 0;
+        const readback = {};
+        let altitudeChangeString = '';
+
+        if (isClimbingOrDescending) {
+            altitudeChangeString = 'leaving ';
+        } else {
+            altitudeChangeString = 'at ';
+        }
```

Comment:
No need for the else here. And stylistically, if we were going to have them say `at 6500`, we would set the variable to `at ` in the initial let declaration, thus avoiding the need for the `else` branch.